src/zenml/cli/secret.py

```diff
@@ -308,16 +308,24 @@ def delete_secret_set(
 
 @secret.command("cleanup")
 @click.option(
-    "--force",
-    "-f",
+    "--yes",
+    "-y",
     "force",
     is_flag=True,
     help="Force the deletion of all secrets",
     type=click.BOOL,
 )
+@click.option(
+    "--force",
+    "-f",
+    "old_force",
+    is_flag=True,
+    help="DEPRECATED: Force the deletion of all secrets",
```

Comment:
I would say "Use `-y` instead"

Reply:
I've made PRs in the ZenBytes and ZenFiles repository with both @fa9r and @ayush714 added as reviewers, so they should be aware of this change.

Reply:
My latest commit adds this message as well.